solutions/USACO open18-talent-show.mdx

```diff
@@ -0,0 +1,35 @@
+---
+id: usaco-839
+source: USACO Gold 2018 Open
+title: Talent Show
+author: Arpan Banerjee
+---
+
+## Solution 1
+Instead of using binary search as the official solution does, an simple knapsack can be done. An array of talents can be kept, where `talent[i]` is the minimum weight required to create a talent sum of i. To make sure each cow is used at most once and the order of the cows doesn't matter, the indices of `talent` are iterated though from greatest to least, and the `cows` are iterated though in the outer loop; the implementation should make this clear. This maximizes the ratio $t/w$ as desired. Then, the maximum ratio of $t_i / w_i$ can be easily found by going through `talent` and ignoring any $w_i$ that is less than $W$. 
```

Comment:
this is a wrong solution, I just stress-tested your code against the one in the editorial (I guess the test cases on usaco was weak).

heres the test that got different outputs

```
10 40
9 28
8 2
6 19
9 32
4 12
9 12
4 36
1 32
2 7
3 6
```
your code outputs `4071` while the code in the editorial outputs `4150` (a better answer)